lib/taskana-core-test/src/test/java/acceptance/task/claim/ClaimTaskAccTest.java

```diff
@@ -0,0 +1,389 @@
+        .isEqualTo(claimedTask.getClaimed());
+    assertThat(claimedTask.isRead()).isTrue();
+    assertThat(claimedTask.getOwner()).isEqualTo("user-1-2");
+  }
+
+  @WithAccessId(user = "user-1-2")
+  @Test
+  void should_ThrowException_When_TaskIsAlreadyClaimedByAnotherUser() throws Exception {
+    Task task =
+        TaskBuilder.newTask()
+            .state(TaskState.CLAIMED)
+            .claimed(Instant.now())
+            .owner("user-1-1")
+            .classificationSummary(defaultClassificationSummary)
+            .workbasketSummary(defaultWorkbasketSummary)
+            .primaryObjRef(defaultObjectReference)
+            .buildAndStore(taskService);
+
+    ThrowingCallable call = () -> taskService.claim(task.getId());
+    InvalidOwnerException e = catchThrowableOfType(call, InvalidOwnerException.class);
+    assertThat(e.getCurrentUserId()).isEqualTo("user-1-2");
+    assertThat(e.getTaskId()).isEqualTo(task.getId());
+  }
+
+  @WithAccessId(user = "user-1-2")
+  @Test
+  void should_ThrowException_When_TaskIsAlreadyInReviewByAnotherUser() throws Exception {
+    Task task =
+        TaskBuilder.newTask()
+            .state(TaskState.IN_REVIEW)
+            .claimed(Instant.now())
+            .owner("user-1-1")
+            .classificationSummary(defaultClassificationSummary)
+            .workbasketSummary(defaultWorkbasketSummary)
+            .primaryObjRef(defaultObjectReference)
+            .buildAndStore(taskService);
+
+    ThrowingCallable call = () -> taskService.claim(task.getId());
+    InvalidOwnerException e = catchThrowableOfType(call, InvalidOwnerException.class);
+    assertThat(e.getCurrentUserId()).isEqualTo("user-1-2");
+    assertThat(e.getTaskId()).isEqualTo(task.getId());
+  }
+
+  @WithAccessId(user = "user-1-2")
+  @Test
+  void should_ClaimTask_When_AlreadyClaimedByCaller() throws Exception {
+    Task task =
+        TaskBuilder.newTask()
+            .state(TaskState.CLAIMED)
+            .claimed(Instant.now())
+            .owner("user-1-2")
+            .classificationSummary(defaultClassificationSummary)
+            .workbasketSummary(defaultWorkbasketSummary)
+            .primaryObjRef(defaultObjectReference)
+            .buildAndStore(taskService);
+
+    assertThat(task.getState()).isSameAs(TaskState.CLAIMED);
+    assertThatCode(() -> taskService.claim(task.getId())).doesNotThrowAnyException();
+  }
+
+  @WithAccessId(user = "user-1-2")
+  @Test
+  void should_ClaimTask_When_AlreadyInReviewByCaller() throws Exception {
+    Task task =
+        TaskBuilder.newTask()
+            .state(TaskState.IN_REVIEW)
+            .claimed(Instant.now())
+            .owner("user-1-2")
+            .classificationSummary(defaultClassificationSummary)
+            .workbasketSummary(defaultWorkbasketSummary)
+            .primaryObjRef(defaultObjectReference)
+            .buildAndStore(taskService);
+
+    assertThat(task.getState()).isSameAs(TaskState.IN_REVIEW);
```

Comment:
You don't need this assertion, because you set the state in the lines above. Could you remove it?